.gitignore

```diff
@@ -1,8 +0,0 @@
-# Mac DS_Store Files
```

Comment:
You have deleted our `.gitignore` file that is inside the `Dufuna-Fem` folder. Kindly replace what you delete back